guides/common/modules/proc_importing-the-katello-root-ca-certificate.adoc

```diff
@@ -1,8 +1,10 @@
-[id="Installing_the_Katello_Root_CA_Certificate_{context}"]
-= Installing the Katello Root CA Certificate
+[id="Importing_the_Katello_Root_CA_Certificate_{context}"]
+= Importing the Katello Root CA Certificate
 
-The first time you log on to {Project}, you might see a warning informing you that you are using the default self-signed certificate and you might not be able to connect this browser to {Project} until the root CA certificate is installed in the browser.
-Use the following procedure to locate the root CA certificate on {Project} and to install it in your browser.
+The first time you log in to {Project}, you might see a warning informing you that you are using the default self-signed certificate and you might not be able to connect this browser to {Project} until the root CA certificate is imported in the browser.
+Use the following procedure to locate the root CA certificate on {Project} and to import it into your browser.
+
+To use the CLI instead of the {ProjectWebUI}, see xref:cli-Importing_the_Katello_Root_CA_Certificate_{context}[CLI Procedure].
```

Comment:
```suggestion
To use the CLI instead of the {ProjectWebUI}, see xref:CLI_Importing_the_Katello_Root_CA_Certificate_{context}[CLI Procedure].
```
Optional.